### PR TITLE
chore(empty-states): Use existing docs for empty state

### PR DESCRIPTION
**Pull request by roggenkemper**
this pr updates the updated issue stream empty state to use existing docs, instead of hardcoded ones, for the existing 4 projects. this change, while not having any impact on what users actually see, will allow for other projects to easily be added while also keeping the instructions as up to date as possible

### CODE REVIEW

File: static/app/components/updatedEmptyState.tsx [hunk 1]
```diff
@@ -143,32 +41,83 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
...
+
+  const currentPlatformKey = project?.platform ?? 'other';
+  const currentPlatform = platforms.find(
+    p => p.id === currentPlatformKey
+  ) as PlatformIntegration;
+
+  const platformPath = getPlatformPath(currentPlatform);
+
+  useEffect(() => {
+    async function getGettingStartedDoc() {
```
Comment (malwilley) [inline static/app/components/updatedEmptyState.tsx:55]: I'm seeing this save useEffect in a few places, would it be possible to make this a common util?

File: static/app/components/updatedEmptyState.tsx [hunk 2]
```diff
@@ -143,32 +41,83 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
...
   ) {
     return null;
   }
 
   const dsn = projectKeys[0].dsn.public;
-  const language = project?.platform === 'node' ? 'javascript' : 'python';
-
+  const {default: docs} = module;
+
+  const docParams: DocsParams<any> = {
```
Comment (malwilley) [inline static/app/components/updatedEmptyState.tsx:83]: Does this have to be `any`?

File: static/app/components/updatedEmptyState.tsx (hunk 1 above)
Comment (roggenkemper) [reply static/app/components/updatedEmptyState.tsx:55]: will merge a separate PR to do this first, then update this one !

File: static/app/components/updatedEmptyState.tsx (hunk 2 above)
Comment (roggenkemper) [reply static/app/components/updatedEmptyState.tsx:83]: it's not great but that's what we have across a bunch of components right now, i can come back and see if it really is necessary after this pr

**Review by malwilley — APPROVED** [review]

**Pull request merged**